crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -1277,6 +1284,16 @@ pub async fn run(
             })?;
             Ok(exit_code)
         }
+        Command::Query { query } => {
+            let query = query.clone();
+            let event = CommandEventBuilder::new("query").with_parent(&root_telemetry);
+            event.track_call();
+            let base = CommandBase::new(cli_args, repo_root, version, color_config);
+
+            let query = query::run(base, event, query).await?;
+
+            Ok(query)
+        }
```

Comment:
Let's add a warn here that this is under development / experimental (wording up to you) just incase anyone finds it in code and wants to play with it early (even though the top level command is hidden for now)